robotouille_input.py

```diff
@@ -33,7 +33,7 @@ def create_action_from_control(env, obs, action, renderer):
         locs = [loc_tuple[2].split(":")[0] if 'station' in loc_tuple[2] else loc_tuple[3].split(":")[0] for loc_tuple in loc_tuples]
         if clicked_station in locs:
             index = locs.index(clicked_station)
-            while 'cook' in str_valid_actions[index] or 'cut' in str_valid_actions[index]:
+            while 'cook(' in str_valid_actions[index] or 'cut(' in str_valid_actions[index] or 'fry(' in str_valid_actions[index] or "fry_cut_item" in str_valid_actions[index]:
```

Comment:
Single quotes around and parentheses after `fry_cut_item` to be consistent with the other actions.

Note for history, because `fry` is technically in a predicate if it includes the `fryer`, we look for `fry(` instead which should be guaranteed to only match the fry action.